modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,110 @@
+
+[NOTE]
+====
+If you use the {product-title} default 503 error code page as a template for your customizations, the headers in the file require an editor than can use CRLF line endings.
+====
+
+.Procedure
+
+. Create a config map named `my-custom-error-code-pages` in the `openshift-config` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-config create configmap my-custom-error-code-pages \
+--from-file=error-page-503.http \
+--from-file=error-page-404.http
+----
+
+. Patch the Ingress Controller to reference the `my-custom-error-code-pages` config map by name:
++
+[source,terminal]
+----
+$ oc patch -n openshift-ingress ingresscontroller/default --patch '{"spec":{"httpErrorCodePages": "my-custom-error-code-pages"}}' --type=merge
+----
++
+The Ingress Operator copies the `my-custom-error-code-pages` config map from the `openshift-config` namespace to the `openshift-ingress` namespace. The Operator names the config map according to the pattern, `<your_ingresscontroller_name>-errorpages`, in the `openshift-ingress` namespace.
+
+. Display the copy:
++
+[source,terminal]
+----
+$ oc get cm default-errorpages -n openshift-ingress
+----
++
+.Example output
+----
+NAME                       DATA   AGE
+default-errorpages         2      25s  <1>
+----
+<1> The example config map name is `default-errorpages` because the `default` Ingress Controller custom resource (CR) was patched.
++
+
+. Confirm that the config map containing the custom error response page mounts on the router volume where the config map key is the filename that has the custom HTTP error code response:
++
+* For 503 custom HTTP custom error code response:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress rsh <router_pod> cat /var/lib/haproxy/conf/error_code_pages/error-page-503.http
+----
++
+* For 404 custom HTTP custom error code response:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress rsh <router_pod> cat /var/lib/haproxy/conf/error_code_pages/error-page-404.http
+----
+
+.Verification
+
+Verify your custom error code http response:
+
+. Create a test project and application:
++
+[source,terminal]
+----
+ $ oc new-project test-ingress
+----
++
+[source,terminal]
+----
+$ oc new-app django-psql-example
+----
+. For 503 custom http error code response:
+.. Stop all the pods for the application.
+.. Run the following curl command or visit the route hostname in the browser:
++
+[source,terminal]
+----
+$ curl -vk <route_hostname>
+----
++
+. For 404 custom http error code response:
+.. Visit a non-existent route or an incorrect route.
+.. Run the following curl command or visit the route hostname in the browser:
++
+[source,terminal]
+----
+$ curl -vk <route_hostname>
+----
+
+Check if the `errorfile` attribute is properly in the `haproxy.config` file:
```

Comment:
Do you want this unnumbered?   Or did you intend to add `. Check...`?

Reply:
I was toying with having it unnumbered, but i think numbered looks better.